man/man8/zpool-create.8

```diff
@@ -135,12 +138,16 @@ This can be overridden with the
 .Fl m
 option.
 .Pp
-By default all supported features are enabled on the new pool unless the
+By default all supported features (if feature sets are requested using
+.Fl o Ar features
+only supported features from all these sets) are enabled on the new pool
+unless the
 .Fl d
 option is specified.
```

Comment:
How about something like: `By default, all features are enabled on the new pool.  The -d option or the compatibility property (e.g. -o compatibility=2020) can be used to restrict the features that are enabled, so that the pool can be imported on other releases of the ZFS software.`